src/pheromones/hivemind-host/include/hivemind-host/FunctionCallRequestDTO.h

```diff
@@ -9,7 +9,7 @@
 class FunctionCallRequestDTO {
 
   public:
-    static constexpr uint8_t FUNCTION_CALL_ARGUMENTS_MAX_LENGTH = 16;
+    static constexpr uint8_t FUNCTION_CALL_ARGUMENTS_MAX_LENGTH = 8;
```

Comment:
It would really qualify as a nice to have, but maybe we could make this templated via CMake (as well as it's counterpart in pheromone) that ways both defines are always in sync and we won't end up having any run time errors